Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -50,23 +50,23 @@ protected
     "Lower value for the output"
     annotation (Placement(transformation(extent={{-80,20},{-60,40}})));
   Buildings.Controls.OBC.CDL.Reals.Subtract conErr
-    "Control error (set point - measurement)"
+    "Control error (measurement - set point)"
```

Comment:
In `CDL.Reals.PID` and other PID controller variants, the control error is (set point - measurement). See [here](https://github.com/lbl-srg/modelica-buildings/blob/b282060ebdb814b4b1c2ecde566843d50ce05493/Buildings/Controls/OBC/CDL/Reals/PID.mo#L56).
I see that the output `yErr` from this block becomes input to the block `PIDWithAutotuning.SystemIdentification.FirstOrderTimedelayed.ControlProcessModel`, so for this block, is it expecting `set point - measurement` or `measurement - setpoint`?

Reply:
I removed the <code>yErr</code> and define a new variable called <code>yDiff</code>. <code>yDiff</code> = <code>yErr</code> if <code>reverseActing</code> is true; otherwise <code>yDiff = -yErr</code>.